source/MaterialXView/Editor.cpp

```diff
@@ -374,7 +398,7 @@ void PropertyEditor::addItemToForm(const mx::UIPropertyItem& item, const std::st
             {
                 // Transform sRGBA color picker value to linear space for writing to material
                 mx::Color3 linearCol = mx::Color3(c.r(), c.g(), c.b()).srgbToLinear();
-                mx::Vector3 v(linearCol[0], linearCol[1], linearCol[2]);
+                mx::Vector4 v(linearCol[0], linearCol[1], linearCol[2], c.a());
```

Comment:
Additionally, I realized that in my [previous PR](https://github.com/AcademySoftwareFoundation/MaterialX/pull/2394) to handle color space transformations, I erroneously concatenated all 4D color vectors to 3D when writing to material. This is my mistake, and is fixed in this PR.